examples/freertos/explorer_board/src/gpio_ctrl/gpio_ctrl.c

```diff
@@ -13,8 +13,8 @@
 /* Library headers */
 
 /* App headers */
-#include "app_conf.h"
-#include "example_pipeline/example_pipeline.h"
+#include "../app_conf.h"
```

Comment:
Could you return this to a non relative path, as it will prevent portability by forcing a specific file structure.